### PR TITLE
Adjust one-sentence description under main heading.

### LINKED ISSUE #7: Decide on Single Sentence Description for front page header
A single descriptive sentence or one-liner that describes US-RSE and our mission and organization would be a great design feature. 
We just need to decide on what it should be. I think it's a larger discussion than just the website wg.  

The one there now "Building software and community in support of research." is a great place holder!

### PR DESCRIPTION
**Pull request by mtbc**
## Description
Changes the text under the landing page's main heading for the non-mobile view.

## Motivation and Context
Fixes #7.

## Checklist:
- [ ] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @usrse-maintainers

### CODE REVIEW

**Comment by mtbc** [comment]
Wondering if the styling needs adjustment, e.g., reduce text size? If so, curious how best to approach that via the CSS.

**Comment by vsoch** [comment]
Let me take a look!

**Comment by vsoch** [comment]
Huh, yeah it seems kind of long.

image

"connecting the people" I like, but "that make research software happen" seems kind of long and mumbled. Why not just:

image

**Comment by cosden** [comment]
There seems to be two related, but separate issues/questions. The first is the slogan (for lack of a better word) and the second is the location, size, format, etc of the text. I'd like to try and settle on the first, and then do what we can to creatively make it look good.

**Comment by cosden** [comment]
Oh, and I had another idea about this that I've been thinking  about: if we can't decide on a perfect single sentence (there are lots of good ones), maybe we don't have to. 

How hard would it be to have a handful of good ones, and then one gets picked (either randomly, or cycled through at some interval) to be displayed each time?

**Comment by mtbc** [comment]
I was looking not to exclude. e.g., the devops folks supporting the CI and release, etc. - i.e. not just cover those who develop the software itself but those who contribute differently. I guess I'll think harder once I've woken up some more!

**Comment by mtbc** [comment]
I'm happy to experiment with size and format of the current but I'd be grateful for any guidance on how to do that while trampling minimally over the current CSS setup and having bad side-effects!

**Comment by mtbc** [comment]
> the location, size, format, etc

How's the latest commit?

**Comment by vsoch** [comment]
Can you note preview locally and take a screenshot?

**Comment by mtbc** [comment]
How do I note preview locally?

header

**Comment by vsoch** [comment]
What do you mean how do you note? That preview looks good! I can be less picky about the actual sentence - if you like it and @cosden does too I'm good :)

**Comment by cosden** [comment]
I can't say I'm 100%, but it's absolutely good enough for now. So I say we go with it, but leave the door open for future discussion changes. I like the phrase and the formatting. Good call on the single line and italics. 
But I have two thoughts/questions:
1. Can we capitalize the word "Connecting"? 
2. To me it feels like even though it's not a complete sentence that it should have a period at the end. This with the capitalization would make it a complete (and more powerful) thought. What do others think?

I'll add them as suggestions. 

Sorry if this seems overly picky. I put a lot of stock into things like this that will contribute to people's first impression. It's really worth the effort.

File: _config.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ paginate: 5
 paginate_path: "/news/page:num/"
 
 hero_line_one: The United States Research Software Engineer Assocation
-hero_line_two: Building software and community
+hero_line_two: connecting the people who make research software happen
```
Comment (cosden) [inline _config.yml:26]:
```suggestion
hero_line_two: Connecting the people who make research software happen.
```

**Comment by mtbc** [comment]
> Can you note preview locally and take a screenshot?

> What do you mean how do you note? 

Oh, you weren't proposing two separate actions? :smile:

**Comment by mtbc** [comment]
header

(I thought I already pasted the update, clearly I failed somehow!)

**Review by cosden — APPROVED** [review]
Looks great! I say we merge, move on, and come back to it later if needed.

**Review by vsoch — APPROVED** [review]

**Pull request merged**